engine/wwtlib/Imageset.cs

```diff
@@ -860,7 +860,7 @@ public double ViewCenterY
                 }
                 if (Levels > 0)
                 {
-                    return CenterY + OffsetY;
+                    return CenterY;
```

Comment:
Same commentary as in `OffsetX`. Here too, based on my own documentation, I think that the sign should be correct.